nncf/experimental/tensor/functions/numeric.py

```diff
@@ -522,3 +522,65 @@ def size(a: Tensor) -> int:
     :return: The size of the input tensor.
     """
     return size(a.data)
+
+
+@functools.singledispatch
+@tensor_guard
+def quantile(a: Tensor, q: float) -> Tensor:
```

Comment:
Please add tests for new function in https://github.com/alexsu52/nncf/blob/develop/tests/shared/test_templates/template_test_nncf_tensor.py